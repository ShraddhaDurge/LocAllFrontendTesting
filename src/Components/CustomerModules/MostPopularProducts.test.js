import React from 'react';
import renderer from 'react-test-renderer';
import MostPopularProducts from './MostPopularProducts';

it('MostPopularProducts', () => {
    const tree=renderer.create(<MostPopularProducts />).toJSON();
    expect(tree).toMatchSnapshot();
})