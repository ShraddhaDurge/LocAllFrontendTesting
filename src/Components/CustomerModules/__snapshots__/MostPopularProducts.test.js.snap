// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MostPopularProducts 1`] = `
<div
  className="MuiGrid-root"
>
  <h5
    className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Most Popular Products
  </h5>
  <div
    className="react-multi-carousel-list  "
  >
    <ul
      className="react-multi-carousel-track "
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseMove={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "overflow": "unset",
          "transform": "translate3d(0px,0,0)",
          "transition": "none",
        }
      }
    />
  </div>
</div>
`;
